src/parser/esprima_cpp/esprima.cpp

```diff
@@ -5752,14 +5717,7 @@ class Parser {
 
         MetaNode endNode = this->createNode();
         this->currentScopeContext->m_bodyEndLOC.index = endNode.index;
-#if !(defined NDEBUG) || defined ESCARGOT_DEBUGGER
-        this->currentScopeContext->m_bodyEndLOC.line = endNode.line;
-        this->currentScopeContext->m_bodyEndLOC.column = endNode.column;
-#endif
```

Comment:
Why did you remove these lines?
ExtendedNodeLOC is no longer used for debugger?
Then please enable these lines for debug build since ExtendedNodeLOC is also used in `debug` mode.
(I'm wondering that ExtendedNodeLOC is actually necessary for debugger now)

Reply:
Opps, I didn't notice the `!(defined NDEBUG) ` part of the guard. I'm going to revert it.